sbncode/EventGenerator/MeVPrtl/Tools/IMesonGen.h

```diff
@@ -0,0 +1,47 @@
+/**
+ *  @file   IMesonGen.h
+ *
+ *  @brief  This provides an art tool interface definition for tools which can create
+ *          fake particles to overlay onto input daq fragments during decoding
+ *
+ *  @author grayputnam@uchicago.edu
+ *
+ */
+#ifndef IMesonGen_h
+#define IMesonGen_h
+
+// Framework Includes
+#include "fhiclcpp/ParameterSet.h"
+#include "art/Framework/Principal/Event.h"
+
+#include "nusimdata/SimulationBase/MCFlux.h"
+
+#include "IMeVPrtlStage.h"
+
+// Algorithm includes
+
+//------------------------------------------------------------------------------------------------------------------------------------------
+
+namespace evgen
+{
+namespace ldm {
+/**
+ *  @brief  IMesonGen interface class definiton
```

Comment:
Add a comment here to say what's the purpose of this interface. 
Is it only to define GetNext and GetPOT? It is  enough, but please state it